pkg/store/acceptance_test.go

```diff
@@ -58,6 +58,12 @@ func testLabelAPIs(t *testing.T, startStore func(extLset labels.Labels, append f
 				{start: timestamp.FromTime(minTime), end: timestamp.FromTime(maxTime), expectErr: errors.New("rpc error: code = InvalidArgument desc = label name parameter cannot be empty")},
 				{start: timestamp.FromTime(minTime), end: timestamp.FromTime(maxTime), label: "foo"},
 				{start: timestamp.FromTime(minTime), end: timestamp.FromTime(maxTime), label: "region", expectedValues: []string{"eu-west"}}, // External labels should be visible.
```

Comment:
It's uncertain whether this test should pass? There are no series, but external label is returned anyway? I didn't touch it because the comment says it should be visible. In case you agree that it's wrong too, I can tweak the PR to account for that (we would need to check `len(vals) > 0` returned from `storepb.LabelValuesResponse`).